src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -11319,27 +11341,51 @@ public static LocalState UnreachableState(Variables variables)
                 return CreateReachableOrUnreachableState(variables, reachable: false);
             }
 
+            public static LocalState ReachableStateWithNotNulls(Variables variables)
+            {
+                var container = variables.Container is null ?
+                    null :
+                    new Boxed(ReachableStateWithNotNulls(variables.Container));
+
+                int capacity = variables.NextAvailableIndex;
+                return new LocalState(variables.Id, container, createBitVectorWithNotNulls(capacity, reachable: true));
+
+                static BitVector createBitVectorWithNotNulls(int capacity, bool reachable)
```

Comment:
>createBitVectorWithNotNulls [](http://example.com/codeflow?start=33&length=27)

Rather than creating a separate method, consider updating `CreateBitVector()` below by adding an `bool initializeWithNotNulls` parameter.

Reply:
The benefit of keeping this logic local is that no other method can unintentionally initialize with NotNulls. The NotNull initialization is the exception and it is now explicit.
I think this benefit outweighs the benefit of factoring two lines.

Reply:
It looks like the reachable parameter will always be `true`. Should the parameter be removed?